### PR TITLE
Adding mdsapt

### PR DESCRIPTION
**Pull request by ifd3f**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~~[ ] If static libraries are linked in, the license of the static library is packaged.~~ N/A
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ifd3f** [comment]
@ALescoulie do you want to be listed as a maintainer?

**Comment by ALescoulie** [comment]
> @ALescoulie do you want to be listed as a maintainer?

yes I'd like to be a maintainer

**Comment by ifd3f** [comment]
@conda-forge/staged-recipes, ready for review

**Comment by ifd3f** [comment]
@conda-forge/help-python

File: recipes/mdsapt/meta.yaml
Comment (xhochy) [inline recipes/mdsapt/meta.yaml:13]: This can probably be noarch.
```suggestion
  noarch: python
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/mdsapt/meta.yaml
Comment (ifd3f) [reply recipes/mdsapt/meta.yaml:13]: No, it can't, because it's not supported on windows, and [the bot complained that it can't be noarch because of that](https://github.com/conda-forge/staged-recipes/pull/24782#issuecomment-1953272625). However, I should change it to support osx, because it does.

File: recipes/mdsapt/meta.yaml
Comment (xhochy) [inline recipes/mdsapt/meta.yaml:13]: What does it mean that it isn't supported on Windows? If a dependency is missing and CI is failing because of that, that's OK. If the actual code isn't working, then you should add a run dependency on `__unix`

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/mdsapt/meta.yaml
Comment (ifd3f) [reply recipes/mdsapt/meta.yaml:13]: The project doesn't support Windows because its transitive dependencies don't, and it's not tested or built on Windows either. The CI fails on Windows if I get rid of the selector, anyways.

Comment (xhochy) [reply recipes/mdsapt/meta.yaml:13]: This then sounds like a case for making this a noarch: python package. Tge failing CI is OK. This means it is not installable on Windows but we still only have a single build. Thisnis is the preferred way to solve the issue

Comment (ifd3f) [reply recipes/mdsapt/meta.yaml:13]: Oh, alright then. I guess I'll do that

**Comment by ifd3f** [comment]
Hi @xhochy , I made the noarch change, could you look over this again?

File: recipes/mdsapt/meta.yaml
Comment (xhochy) [inline recipes/mdsapt/meta.yaml:12]: You need to specify the entry_points here.

File: recipes/mdsapt/meta.yaml
Comment (xhochy) [inline recipes/mdsapt/meta.yaml:19]:
```suggestion
    - python >=3.9
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/mdsapt/meta.yaml
Comment (xhochy) [inline recipes/mdsapt/meta.yaml:2]: Once we have a release, I'm happy to merge. This looks good

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**
